### DISCUSSION TITLE
library logo

### DISCUSSION THREAD

**Discussion by probablyArth** [sanny-io/react-appwrite#21] [Polls]
leave your thoughts below 👇
| Logo             |  # |
:-------------------------:|:-------------------------:
![react-appwrite-2](https://user-images.githubusercontent.com/61962871/229100433-e6fc5e0a-80dc-43e0-859b-89500f00e799.png) | 1
![react-appwrite-1](https://user-images.githubusercontent.com/61962871/229100441-ecc684ce-00ad-40db-8f28-7aacaa49e290.png) | 2

**Comment by probablyArth** [comment]
@sanny-io @dishwasher-detergent

**Comment by sanny-io** [comment]
I vote for 2. Did you make these yourself? They're pretty good haha.

**Reply by probablyArth** [reply]
Yes thanks haha, I vote for two aswell ;)
